src/routes/jira/jira-router.ts

```diff
@@ -5,13 +5,16 @@ import { JiraAtlassianConnectGet } from "./jira-atlassian-connect-get";
 import { JiraEventsRouter } from "./events/jira-events-router";
 import { JiraSelectVersionRouter } from "./server/jira-select-version-router";
 import { JiraContextJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
-import { JiraServerUrlRouter } from "./server/jira-server-url-router";
+import { JiraServerUrlRouter } from "routes/jira/server/jira-server-url-router";
+import { JiraAppCreationRouter } from "./server/jira-app-creation-router";
 
 export const JiraRouter = Router();
 
 JiraRouter.get("/atlassian-connect.json", JiraAtlassianConnectGet);
 JiraRouter.use("/configuration", JiraConfigurationRouter);
+// TODO - add csrf middleware
```

Comment:
To be tackled as part of [ARC 1474](https://softwareteams.atlassian.net/browse/ARC-1474)